docs/release/trg-8/trg-8-06.md

```diff
@@ -0,0 +1,33 @@
+---
+title: TRG 8.06 -  SNYK
+---
+
+| Status | Created     | Post-History                         |
+|--------|-------------|--------------------------------------|
+| Active | 21-Feb-2024 | Initial release                      |
+
+## Why
+
+Our primary aim is to improve security and define best practices across the Tractus-X ecosystem.
+
+## Description
+
+A security scanning toolchain is a collection of tools and processes that are used to scan software applications for security vulnerabilities. These tools can be used at various stages of the software development lifecycle (**SDLC**), from development to deployment and beyond.
+
+### SNYK
+
+To integrate SNYK with your GitHub repository, you need to follow these steps:
+
+- Login to SNYK using your GitHub account
+- Go to the Integrations page in your SNYK account and click **Connect to GitHub**

```

Comment:
As far as I know the connection to GH can only be set up by the sec team, as they are Snyk admins but this (especially also the part below regarding 'To import GitHub Projects') implies that everybody could set up the connection on its own.